Sources/RuleBuilder/Dictionary.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Dictionary.swift
+//
+//
+//  Created by Yume on 2022/8/5.
+//
+
+import Foundation
+
+public struct Dictionary: Text, CodeText {
+    // MARK: Lifecycle
```

Comment:
Do we consider a different name for `Dictionary`?

Because `Foundation.Dictionary` is pretty common.

Reply:
How about with prefix `Starlark` -> `StarlarkDictionary`.

Reply:
Looks good to me!

Please feel free to move forward as we can refactor/rename them in the future easily.